index.bs

```diff
@@ -715,6 +715,26 @@ A <dfn>trigger spec</dfn> is a [=struct=] with the following items:
 A <dfn>trigger spec map</dfn> is a [=map=] whose keys are unsigned 32-bit
 integers and values are [=trigger specs=].
 
+To <dfn>find a matching trigger spec</dfn> given an [=attribution source=]
+|source| and an unsigned 64-bit integer |triggerData|:
+
+1. Let |specs| be |source|'s [=attribution source/trigger specs=].
+1. If |source|'s [=attribution source/trigger-data matching mode=] is:
+    <dl class="switch">
+    : "<code>[=trigger-data matching mode/exact=]</code>"
+    :: Run the following steps:
+        1. If |specs|[|triggerData|] [=map/exists=], return its [=map/entry=].
+        1. Return null.
+    : "<code>[=trigger-data matching mode/modulus=]</code>"
+    :: Run the following steps:
+        1. If |specs| [=map/is empty=], return null.
+        1. Let |keys| be |specs|'s [=map/get the keys|keys=].
+        1. Let |index| be the remainder when dividing |triggerData| by |keys|'s
+            [=set/size=].
+        1. Return the [=map/entry=] for |specs|[|keys|[|index|]].
```

Comment:
Should this also address the possibility that an entry does not exist?

Reply:
The entry is guaranteed to exist as a result of the preceding remainder operation and emptiness check.

Reply:
Oh, oops, my bad.